Sample Applications/DataBindingDemo/App.xaml

```diff
@@ -71,7 +71,7 @@
                         <DataTrigger.Value>
                             <local:SpecialFeatures>Highlight</local:SpecialFeatures>
                         </DataTrigger.Value>
-                        <Setter Property="BorderBrush" Value="Orange" TargetName="border" />
+                        <Setter Property="BorderBrush" Value="#C67E0A" TargetName="border" />
```

Comment:
Rather than using RGB code, we can use predefined colors like Indigo, Green, OrangeRed, etc. It will maintain uniformity with the rest of the code.

Reply:
Updated the color to OrangeRed